packages/react-vapor/src/utils/JSXUtils.spec.tsx

```diff
@@ -25,4 +27,21 @@ describe('JSXUtils', () => {
             ).toBe("Hello there! Can't you see me? I can.");
         });
     });
+
+    describe('addClassNameToChildren', () => {
+        it('should not add any class if the children is not a react element', () => {
+            const myElement = 'a string is not a react element';
+            expect(addClassNameToChildren(myElement, 'new-class')).toContain(myElement);

```

Comment:
Would it work with toBe instead of toContain?
```suggestion
            expect(addClassNameToChildren(myElement, 'new-class')).toBe(myElement);
```

Reply:
`addClassNameToChildren` returns an array